app/decorators/case/ico/sar_decorator.rb

```diff
@@ -1,7 +1,7 @@
 class Case::ICO::SARDecorator < Case::ICO::BaseDecorator
 
   def pretty_type
-    'ICO appeal for SAR case'
+    'ICO appeal(SAR)'
```

Comment:
The design calls for this to be:

> ICO Appeal (SAR)

Note capital 'Appeal' and the extra space. Might need to confirm with Mia whether the capital "A" is actually correct.

Reply:
Checking this with Mia. ATM it matches what we have for FOI i.e "ICO appeal(FOI)"